codemcp/testing.py

```
@@ -107,7 +107,10 @@ async def setup_repository(self):
         By default, it initializes a git repository and creates an initial commit.
         """
         # Initialize and configure git
-        await self.git_run(["init", "-b", "main"])
+        try:
+            await self.git_run(["init", "-b", "main"])
+        except subprocess.CalledProcessError:
+            self.fail("git version is too old for tests! Please install a newer version of git.")
```

better to also report the exception too, probably